Tasks/DownloadPackageV1/task.json

```diff
@@ -0,0 +1,188 @@
+{
+    "id": "e9e6165a-f8e3-4341-838f-cde7a2f23195",
```

Comment:
8d6e8f7e-267d-442d-8c92-1f586864c62f - needs to be the same as DownloadPackageV0